frontend/src/CopyValue.js

```diff
@@ -17,15 +17,21 @@ const Hover = styled.span`
 `;
 
 class CopyValue extends Component {
+  constructor(props) {
```

Comment:
If initial state does not depend on props, just remove constructor and set `state` directly

```js
state = { isStricken: false }
```